app/controllers/topics-controller.js

```diff
@@ -0,0 +1,10 @@
+const {selectTopics} = require('../models/topics-model')
+
+exports.getTopics = (req, res, next) =>{
+    selectTopics().then(({rows}) =>{
+        res.status(200).send(rows)
+    })
+    .catch((err)=>{
+        next(err)
+    })
```

Comment:
Really nice to see a catch block included here, just incase something unexpected happens